third_party/blink/renderer/core/brave_page_graph/page_graph.cc

```diff
@@ -951,10 +952,10 @@ void PageGraph::GenerateReportForNode(const blink::DOMNodeId node_id,
 
   for (const GraphNode* succ : successors) {
     ItemName item_name = succ->GetItemName();
-    if (item_name.find("resource #") == 0) {
+    if (!item_name.StartsWith("resource #")) {
```

Comment:
why negation? it should be `if (item_name.StartsWith("resource #")) {`.

Reply:
Done. I had assumed I was replacing `count` there. Thanks for noticing that.